force-app/main/default/classes/TestUtils.cls

```diff
@@ -0,0 +1,61 @@
+public with sharing class TestUtils {
```

Comment:
in case you are gonna use this class as test factory only, meaning only for test classes then it's better to annotate this class as @isTest -> this way it will be not needed to cover this class by tests as with any regular Apex classes